static/app/components/feedback/feedbackOnboarding/feedbackOnboardingLayout.tsx

```diff
@@ -72,6 +73,7 @@ export function FeedbackOnboardingLayout({
   return (
     <AuthTokenGeneratorProvider projectSlug={projectSlug}>
       <Wrapper>
+        <Header>{introduction && <Introduction>{introduction}</Introduction>}</Header>
```

Comment:
Seems like an empty `<Header>` will be rendered if there's no `introduction`. The css looks like it'll be ok in that case (should result in a 0px tall div). just pointing it out.

Reply:
ahh let me wrap the whole thing in the conditional